OpenRA.Mods.Common/Graphics/DefaultSpriteSequence.cs

```diff
@@ -122,32 +123,97 @@ public FileNotFoundSequence(FileNotFoundException exception)
 		float ISpriteSequence.GetAlpha(int frame) { throw exception; }
 	}
 
+	[Desc("This is a default sprite sequence.")]
 	public class DefaultSpriteSequence : ISpriteSequence
 	{
 		static readonly WDist DefaultShadowSpriteZOffset = new WDist(-5);
 		protected Sprite[] sprites;
 		readonly bool reverseFacings, transpose;
 		readonly string sequence;
-		readonly float[] alpha;
 
 		protected readonly ISpriteSequenceLoader Loader;
 
 		public string Name { get; private set; }
+
+		[Desc("")]
 		public int Start { get; private set; }
-		public int Length { get; private set; }
+
+		[Desc("")]
+		public int Length { get; private set; } = 1;
+
+		[Desc("")]
```

Comment:
Especially this one is far from easy to understand, it definitely needs documentation.